### DISCUSSION TITLE
Any plan for a 0.5.3, or waiting for 0.6?

### DISCUSSION THREAD

**Discussion by Nairou** [emersion/maddy#452] [General]
I'm eager for the IMAP IDLE fix :smile: 
I'm hesitant to build from source, figured I'd see if there was a known timeline. Thanks!

**Comment by foxcpp** [comment]
There is no fixed timeline for releases. I currently have little to no time to work on maddy, unfortunately, but I will do my best to tag 0.5.3 release this week.

**Reply by foxcpp** [reply]
I just tagged 0.5.3: https://github.com/foxcpp/maddy/releases/tag/v0.5.3
